meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: tomopy
-  version: '0.1.15'
+  version: '1.0.0'
```

Comment:
Just realized we set this up in docs, so I increased first one: The minor version is incremented for releases which add new, but backward-compatible, API features, and the major version is incremented for API changes which are not backward-compatible.